osquery/core/posix/process.cpp

```diff
@@ -127,4 +129,20 @@ std::shared_ptr<PlatformProcess> PlatformProcess::launchExtension(
 
   return std::make_shared<PlatformProcess>(ext_pid);
 }
+
+std::shared_ptr<PlatformProcess> PlatformProcess::launchPythonScript(
+    const std::string& args) {
+  std::shared_ptr<PlatformProcess> process;
+
+  int process_pid = ::fork();
+  if (process_pid == 0) {
+    // Start a Python script
+    ::execlp("sh", "sh", "-c", args.c_str(), nullptr);
```

Comment:
This function is called `launchPythonScript`, is it worth us statically coding/deriving the path to python as opposed to executing whatever the caller hands us, similar to how it's being done in the windows implementation?